app/queues/notifications/certificate-expiration-monitor.server.ts

```diff
@@ -0,0 +1,53 @@
+import { Queue } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { prisma } from '~/db.server';
+import {
+  startCertificateExpirationMonitorWorker,
+  certExpirationQueueName,
+} from './certificate-expiration-monitor-worker.server';
+
+const { NOTIFICATIONS_REPEAT_PATTERN } = process.env;
+
+// fetch records with by status
+const getCertificatesByExpiration = async () => {
+  let monthFromToday = new Date();
+  monthFromToday.setMonth(monthFromToday.getMonth() + 1);
+  return await prisma.certificate.findMany({
```

Comment:
You don't need `async` and `await` in this code, because you aren't consuming the result of the `Promise`.  You can return the `Promise` directly and let the caller `await` it.